File: java/java-practise/src/main/java/day/one/ReverseString.java
package day.one;

public class ReverseString {

    public static String toReverse(String textToReverse){
        String reversedText = "";
        for(int i = textToReverse.length() - 1; i >= 0; i--){
            reversedText += textToReverse.charAt(i);
        }
        return reversedText;
    }
}

File: scharp/day/one/ReverseString/ReverseString.cs
using System.Text;

namespace ReverseString
{
    public class ReverseString
    {
        public static String ToReverse(String textToReverse)
        {
            if (textToReverse == null)
            {
                throw new ArgumentNullException(nameof(textToReverse));
            }

            StringBuilder reversedText = new();

            for (int i = textToReverse.Length - 1; i >= 0; i--)
            {
                reversedText.Append(textToReverse[i]);
            }

            return reversedText.ToString();
        }
    }
}
